components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -661,22 +661,23 @@ func (backend ES2Backend) getFiltersQuery(filters map[string][]string, latestOnl
 }
 
 func (backend ES2Backend) getESFieldName(filterType string) string {
+	ESFieldName := filterType
 	switch filterType {
 	case "organization":
-		return "organization_name"
+		ESFieldName = "organization_name"
 	case "chef_server":
-		return "source_fqdn"
+		ESFieldName = "source_fqdn"
 	case "platform":
-		return "platform.name"
+		ESFieldName = "platform.name"
 	case "role":
-		return "roles"
+		ESFieldName = "roles"
 	case "recipe":
-		return "recipes"
+		ESFieldName = "recipes"
 	case "inspec_version":
-		return "version"
-	default:
-		return filterType
+		ESFieldName = "version"
 	}
+
+	return ESFieldName + ".lower"
```

Comment:
The `.lower` addition tells Elasticsearch to search with case insensitive. The `lower` term is defined here https://github.com/chef/automate/blob/master/components/compliance-service/ingest/ingestic/mappings/comp-rep-date.go#L120.